script/tool/lib/src/create_all_packages_app_command.dart

```diff
@@ -229,18 +235,28 @@ dependencies {}
 ''');
     }
 
-    const String lifecycleDependency =
-        "    implementation 'androidx.lifecycle:lifecycle-runtime:2.2.0-rc01'";
+    final String lifecycleDependency = gradleFileIsKotlin
+        ? '    implementation("androidx.lifecycle:lifecycle-runtime:2.2.0-rc01")'
+        : "    implementation 'androidx.lifecycle:lifecycle-runtime:2.2.0-rc01'";
 
     _adjustFile(
       gradleFile,
       replacements: <String, List<String>>{
-        // minSdkVersion 21 is required by camera_android.
-        'minSdkVersion': <String>['minSdkVersion 21'],
-        'compileSdkVersion': <String>['compileSdk 34'],
+        if (gradleFileIsKotlin)
```

Comment:
Formatting nit can you include {} around this case?

Reply:
Collection conditionals can't use braces, unfortunately.